packages/editor-ui/src/components/mixins/workflowHelpers.ts

```diff
@@ -535,11 +553,11 @@ export const workflowHelpers = mixins(
 				let itemIndex = opts?.targetItem?.itemIndex || 0;
 
 				const inputName = 'main';
-				const activeNode = this.$store.getters['ndv/activeNode'];
+				const activeNode = this.ndvStore.activeNode;
 				const workflow = this.getCurrentWorkflow();
-				const workflowRunData = this.$store.getters.getWorkflowRunData as IRunData | null;
-				let parentNode = workflow.getParentNodes(activeNode.name, inputName, 1);
-				const executionData = this.$store.getters.getWorkflowExecution as IExecutionResponse | null;
+				const workflowRunData = this.workflowsStore.getWorkflowRunData as IRunData | null;
+				let parentNode = workflow.getParentNodes(activeNode?.name, inputName, 1);
+				const executionData = this.workflowsStore.getWorkflowExecution as IExecutionResponse | null;
```

Comment:
can remove some of these typecasts I hope now

Reply:
Yup, removing.